src/gui/MainWindow.cpp

```diff
@@ -893,6 +893,10 @@ void MainWindow::updateRecentlyOpenedProjectsMenu()
 		{
 			m_recentlyOpenedProjectsMenu->addAction(
 					embed::getIconPixmap( "project_file" ), *it );
+#ifdef LMMS_BUILD_APPLE
+			m_recentlyOpenedProjectsMenu->actions().last()->setIconVisibleInMenu(false); // Workaround for https://bugreports.qt.io/browse/QTBUG-44565
```

Comment:
I did wonder about URL vs bug number but went with the full URL since that's what we used in #3350.

I'd tend toward still indicating it's a workaround, so would prefer to go with:
```
m_recentlyOpenedProjectsMenu->actions().last()->setIconVisibleInMenu(false);  // QTBUG-44565 workaround
```